pom.xml

```diff
@@ -167,10 +167,10 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
-                <version>3.1</version>
+                <version>3.5</version>
                 <configuration>
-                    <source>1.7</source>
-                    <target>1.7</target>
+                    <source>1.8</source>
```

Comment:
I don't think we want to move the source version to 1.8 unless we move the whole infrastructure (e.g., JDK) to Java 8